src/linux/classes/com/alibaba/wisp/engine/WispEngine.java

```diff
@@ -326,6 +326,17 @@ public boolean enableSocketLock() {
             public StackTraceElement[] getStackTrace(WispTask task) {
                 return task.getStackTrace();
             }
+
+            @Override
+            public void getCpuTime(long[] ids, long[] times) {
+                for (int i = 0; i < ids.length; i++) {
+                    for (WispTask wispTask : WispTask.id2Task.values()) {
+                        if (wispTask.getThreadWrapper().getId() == ids[i]) {
```

Comment:
can `wispTask.getThreadWrapper()` returns `null`?

Reply:
yes.  For exited wispTask this can be null.